### PR TITLE
add blurb about translating the dev wiki

### PR DESCRIPTION
**Pull request by woodiertexas**
---
See preview on Cloudflare Pages: https://preview-117.developer-wiki.pages.dev

### CODE REVIEW

File: wiki/landing-page/en.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This site houses heavily work-in-progress documentation for developing with proj
 Check the sidebar to see what articles are currently available!
 
 If you are looking to contribute, [this is a great place to start as it lists missing and partially completed wiki articles](https://github.com/QuiltMC/developer-wiki/issues/69).
+If you are looking to translate the wiki to your native language, open a translation PR on the [wiki's GitHUb repo](https://github.com/QuiltMC/developer-wiki/compare).
```
Comment (ix0rai) [inline wiki/landing-page/en.md:7]:
```suggestion
If you are looking to translate the wiki to another language, open a translation PR on the [wiki's GitHub repo](https://github.com/QuiltMC/developer-wiki/). There isn't yet documentation on translating, but you can reference pages such as [the getting started page](https://github.com/QuiltMC/developer-wiki/tree/main/wiki/introduction/getting-started) that are already translated to multiple languages.
```

**Review by ix0rai — CHANGES_REQUESTED** [review]

**Review by ix0rai — COMMENTED** [review]
i preferred before https://github.com/QuiltMC/developer-wiki/pull/117/commits/85f3c2717cd6da9a1329dd824a94c0132b96034b, i intend on documenting it

**Comment by woodiertexas** [comment]
> i preferred before [85f3c27](https://github.com/QuiltMC/developer-wiki/commit/85f3c2717cd6da9a1329dd824a94c0132b96034b), i intend on documenting it

"isn't yet documentation..." didn't flow smoothly to me whereas "is currently no documentation..." does flow smoothly to me

**Comment by ix0rai** [comment]
sure, not going to bikeshed

**Pull request merged**